samples/bluetooth/central_uart/src/main.c

```diff
@@ -585,27 +633,19 @@ int main(void)
 		return 0;
 	}
 
-	err = scan_init();
-	if (err != 0) {
-		LOG_ERR("scan_init failed (err %d)", err);
-		return 0;
-	}
-
 	err = nus_client_init();
 	if (err != 0) {
 		LOG_ERR("nus_client_init failed (err %d)", err);
 		return 0;
 	}
 
-	printk("Starting Bluetooth Central UART sample\n");
-
-	err = bt_scan_start(BT_SCAN_TYPE_SCAN_ACTIVE);
+	scan_init();
+	err = scan_start();
 	if (err) {
-		LOG_ERR("Scanning failed to start (err %d)", err);
 		return 0;
 	}
 
-	LOG_INF("Scanning successfully started");
```

Comment:
use logging, not printk

Reply:
I would prefer to leave printk to have output on the uart.
As it was done before my changes.
Logging in on the RTT.

Reply:
I would have preferred to see `Starting Bluetooth Central UART sample` printed on the UART console instead of the RTT console.

Reply:
The user selects where it goes, it can go to RTT, UART, remote NUS device, UDP socket... anywhere with a supported console backend?

Reply:
I agree with @nordicjm, this is a bit confusing no? If the user sets a logging backend they expect all logging to go there. @rugeGerritsen @nordic-krch any thoughts?

Reply:
When RTT logging backend is used then `CONFIG_LOG_PRINTK` is selected so printk will go to RTT anyway. I know that sample code often uses printk, i guess that we don't have any recommendation. Personally, I would prefer logging because it's more flexible and cheaper in terms of resources/time needed to log a message.

Reply:
I agree, using logging would be more consistent.
For some reason, the `central_uart` and `peripheral_uart` samples print a welcome message to the UART console.
My PR doesn't change that, I just moved the line with that message. 
Changing the logging would also require aligning the tests for this sample.

Reply:
The intention of the PR is unrelated to logging/printk, and I'd like it to stay this way. This sample uses UART for forwarding data over BLE, logging uses the RTT backend. Most of the testing in NCS rely on UART output though and expect the welcome log. If there is a strong preference on unifying samples to use logging instead of UART let's do it in a separate PR for all samples and align CI tests.

Reply:
@nordicjm , @carlescufi , this is not VV area of interest. I understand that @nordicjm was triggered to check the prj.conf. I have no idea why this spilled into this elaborate discussion. Please approve the change or give comments on what's wrong in prj.conf.

Reply:
The rule of thumb was that we are using LOG in drivers but printk to print sample data in main.c. I also do not get why should we change this if the PR is totally unrelated to the logging method used.

Reply:
Sure, no problem from my side. @nordicjm could you please remove your nack?